routes/guestsRouter.js

```diff
@@ -0,0 +1,66 @@
+const router = require("express").Router({ mergeParams: true });
+const uuid = require("uuidv4").default;
+const Guest = require("../models/guests");
+const { findGuestById } = require("../middleware");
+
+router.get("/", async (req, res) => {
+	const { weddingId } = req.params;
+	try {
+		const guests = await Guest.find(weddingId);
+		res.status(200).json(guests);
+	} catch (err) {
+		res.status(500).json({
+			error: err.message,
+		});
+	}
+});
+
+router.get("/:id", findGuestById, async (req, res) => {
+	const { guest } = req;
+	res.status(200).json(guest);
+});
+
+router.post("/", async (req, res) => {
+	const { weddingId } = req.params;
+	const guest = req.body;
+	if (Object.entries(guest).length === 0 || !guest.name || !guest.email) {
```

Comment:
Not required but it might make sense to put the guest validation logic on the model so it could be used by any route that allows guest creation.

Reply:
@tinomen how would that look like?